examples/kubernetes/static-provisioning/volume/pv.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: fsx-pv
 spec:
   capacity:
-    storage: 64Gi
+    storage: 0Gi
```

Comment:
Why 0 here? Does this allow the user to write to the volume as normal? If we use 1Gi are we blocked once we hit 1Gi?

Reply:
This was a mistake, and a relic from when I initially thought `0Gi` would work. This should be 1Gi

Reply:
Sounds good. Could you verify we can still write larger than 1Gi when we statically create the PV. I wanna make sure there isn't an internal check which blocks us